hydra-node/exe/hydra-net/Main.hs

```diff
@@ -207,18 +206,17 @@ injectReqSn peer snapshotNumber hydraKeyFile fakeHydraKeyFile = do
       _ -> die "getAdrrInfo failed"
 
   mkApplication sk party tracer =
-    OuroborosApplication $
+    OuroborosApplication
       [ MiniProtocol
           { miniProtocolNum = MiniProtocolNum 42
           , miniProtocolLimits = MiniProtocolLimits{maximumIngressQueue = maxBound}
           , miniProtocolRun =
               InitiatorProtocolOnly
                 ( mkMiniProtocolCbFromPeer
-                    ( \_ ->
-                        ( (contramap TraceSendRecv tracer)
-                        , codecFireForget
-                        , (fireForgetClientPeer $ client tracer sk party)
-                        )
+                    ( const
+                        (contramap TraceSendRecv tracer)
+                    , codecFireForget
+                    , fireForgetClientPeer $ client tracer sk party
```

Comment:
Must: fix this as it does not compile

This binary is not built by CI, so that's why we don't see it red. How did you apply the hlint suggestions? Seems like this broke.